datalad_debian/new_package.py

```diff
@@ -23,7 +23,18 @@
 
 @build_doc
 class NewPackage(Interface):
-    """Create a new package dataset
+    """Create a new package dataset inside of a distribution dataset
+
+    In its final stage, a typical package dataset contains the source files,
+    built binaries, and builder for a Debian package.
```

Comment:
I recommend specifying that it's a "builder subdataset".

Reply:
thx